keylime-ansible-gcp/playbook.yml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Minor nit: Let's give the playbook a better name than just `playbook.yml`. It's almost the equivalent of have a `script.sh` file :)

How about `create_gcp_instance.yml` or something similar.